hangman.php

```diff
@@ -0,0 +1,88 @@
+<?php
+    $word="";
+    function getword()
+    {
+        $file="BandsList.txt";
+        $fopen = fopen($file, "r");
+        $fread = fread($fopen,filesize($file));
+        $split = explode("\n", $fread);
+        $array[] = null;
+        foreach ($split as $string)
```

Comment:
First check if the data is not empty then only a loop should be called so the above code should be like this :
if(!empty($split)) {
foreach ($split as $string)
{
array_push($array,$string);
}
}